src/app/statistics-page/statistics-table/statistics-table.component.ts

```diff
@@ -91,4 +97,25 @@ export class StatisticsTableComponent implements OnInit {
         return of(point.label);
     }
   }
+
+  /**
+   * Defines a dynamic label for the object column
+   * @param reportType
+   */
+  getObjectHeaderLabel(reportType: string): string {
+    switch (reportType) {
+      case 'TotalVisits':
+        return this.translateService.instant('statistics.table.header.TotalVisits');
```

Comment:
I think you can get rid of this entire `switch` if you just append the `reportType` similar to this:

```
return this.translateService.instant('statistics.table.header.' + reportType);
```

That way, if new reports are added, we don't need to add more cases to this switch.